packages/alfa-rules/src/sia-r70/rule.ts

```diff
@@ -0,0 +1,147 @@
+/// <reference lib="dom" />
+
+import { Rule, Diagnostic } from "@siteimprove/alfa-act";
+import { Document, Element, Namespace } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Page } from "@siteimprove/alfa-web";
+import { List } from "@siteimprove/alfa-list";
+import { Iterable } from "@siteimprove/alfa-iterable";
+
+import { expectation } from "../common/expectation";
+import { isIgnored, hasChild } from "../common/predicate";
+import { Group } from "../common/group";
+
+import { isRendered } from "../common/predicate/is-rendered";
+
+import { isDocumentElement } from "../common/predicate/is-document-element";
+
+const { isElement, hasName, hasNamespace } = Element;
+const { and, or, test, not } = Predicate;
+const deprecated = [
+  "applet",
+  "basefont",
+  "bgsound",
+  "big",
+  "blink",
+  "center",
+  "content",
+  "dir",
+  "font",
+  "frame",
+  "frameset",
+  "hgroup",
+  "image",
+  "keygen",
+  "marquee",
+  "menuitem",
+  "nobr",
+  "noembed",
+  "noframes",
+  "plaintext",
+  "rb",
+  "rtc",
+  "shadow",
+  "spacer",
+  "strike",
+  "tt",
+  "xmp",
+];
+export default Rule.Atomic.of<Page, Document, Group<Element>>({
```

Comment:
```suggestion
export default Rule.Atomic.of<Page, Document>({
```
The third type parameter is for questions and shouldn't be needed here.